language/libra-vm/src/errors.rs

```diff
@@ -78,19 +96,30 @@ pub fn convert_normal_prologue_error(status: VMStatus) -> VMStatus {
 /// `UNEXPECTED_ERROR_FROM_KNOWN_MOVE_FUNCTION`
 pub fn convert_normal_success_epilogue_error(status: VMStatus) -> VMStatus {
     match status {
-        VMStatus::MoveAbort(location, code @ EACCOUNT_FROZEN)
-        | VMStatus::MoveAbort(location, code @ EINSUFFICIENT_BALANCE) => {
-            if location != known_locations::account_module_abort() {
+        VMStatus::Executed => VMStatus::Executed,
+        VMStatus::MoveAbort(location, code)
+            if location != known_locations::account_module_abort() =>
+        {
+            let (category, reason) = error_split(code);
+            error!(
+                "[libra_vm] Unexpected success epilogue Move abort: {:?}::{:?} \
+                (Category: {:?} Reason: {:?})",
+                location, code, category, reason
+            );
+            VMStatus::Error(StatusCode::UNEXPECTED_ERROR_FROM_KNOWN_MOVE_FUNCTION)
+        }
+
+        VMStatus::MoveAbort(location, code) => match error_split(code) {
+            (LIMIT_EXCEEDED, ECANT_PAY_GAS_DEPOSIT) => VMStatus::MoveAbort(location, code),
+            (category, reason) => {
```

Comment:
It looks like you dropped handling EACCOUNT_FROZEN here. Was that intentional?

Reply:
I'm continuously frustrated by how little test coverage we have.

I think its a mistake, I'll take a look

Reply:
It looks like someone removed it from the account code but didn't remove it here. Testing might have only fixed this in someone might have realized they needed to clean up the VM, but otherwise its okay I suppose..